openbb_terminal/stocks/tradinghours/tradinghours_controller.py

```diff
@@ -231,3 +237,45 @@ def call_all(self, other_args: List[str]):
         ns_parser = self.parse_known_args_and_warn(parser, other_args)
         if ns_parser:
             bursa_view.display_all()
+
+    @log_start_end(log=logger)
+    def call_holidays(self, other_args: List[str]):
+        """Process 'holidays' command."""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="exchange",
+            description="Select the exchange and year you want see holidays for",
+        )
+        parser.add_argument(
+            "-n",
+            "--name",
+            help="Exchange short name",
+            dest="exchange",
+        )
```

Comment:
this should include the choices here (which will help the autocomplete).  We also use the `metavar="NAME"` convention so that using -h doesn't show a long list.

Reply:
Hi James, I don't understand what the requirement here is. At the moment autocomplete lists all the exchanges for which holidays are displayed.  I'll have to have a look for metavar, not aware of this. cheers

Reply:
There is a 'choices' argument in the add_argument that we can list all available choices, which will add in the auto complete.

Reply:
Hi James, I already set up choices in __init__ with the following lines 
choices["holidays"] = {
                c: None for c in self.all_holiday_exchange_short_names
            }
and it seems to work as you get the full list of available exchanges for the data.
That was how the earlier code in this controller did it.

Reply:
This seems to get stranger, I've just pulled the latest changes from your master and now the autocomplete doesn't seem to work for any of the commands under trading_controller, but when I run with the older codebase they all work just fine.